src/flask_session/dynamodb/dynamodb.py

```diff
@@ -46,8 +77,11 @@ def __init__(
         sid_length: int = Defaults.SESSION_ID_LENGTH,
         serialization_format: str = Defaults.SESSION_SERIALIZATION_FORMAT,
         table_name: str = Defaults.SESSION_DYNAMODB_TABLE,
+        table_exists: Optional[bool] = Defaults.SESSION_DYNAMODB_TABLE_EXISTS,
     ):
 
+        # NOTE: The name client is a bit misleading as we're using the resource API of boto3 as opposed to the service API
```

Comment:
Fair point, the reason for using client is so that all the backends use this, making it easier to see differences in implementation. I think I'm happy with leaving your comment here as a note for future devs.

Reply:
I suspected that. I added it because I work a lot with boto3 and it's very common to use different names for the different SDK-abstraction-levels, this one broke my mental shortcut, I think others may be in the same position ;)

Reply:
Yeah i wouldn't normally be opposed to changing it across the board, but it gets kind of weird with some of them. If anyone has a good PR for it we can see

Reply:
Btw this is merged, I dunno why github doesn't detect that..